sdk/assets/component/train.yml

```diff
@@ -1,21 +1,25 @@
 $schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
-name: my_train_data_component
+name: train_data_component
```

Comment:
I would recommend use different name to avoid conflict

Reply:
make sense